src/pyskindose/table_data/generated_hvl_spekpy_data.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import spekpy as sp  # Note: Not included in PySkinDose,
+import pandas as pd
+import os
+from pathlib import Path
+
+
+def generate_spekpy_data(
+    kvpRange_kV,
+    InherentFiltration_mmAl,
+    AddedFiltrationRange_mmAl,
+    AddedFiltrationRange_mmCu,
+    AddedFiltration_mmAir,
+    DeviceModel,
+    AcquisitionPlane,
+    AnodeAngle_deg,
+    ResultsPath,
+    LabName,
+    FileName,
+    MeasurementDate,
+):
+
+    index = [
+        "kVp_kV",
+        "InherentFiltration_mmAl",
+        "AddedFiltration_mmCu",
+        "AddedFiltration_mmAl",
+        "AddedFiltration_mmAir",
+        "DeviceModel",
+        "AcquisitionPlane",
+        "AnodeAngle_deg",
+        "HVL_mmAl",
+        "LabName",
+        "MeasurementDate",
+    ]
+
+    res = dict()
+    for i in index:
+        res[i] = []
+
+    for kvp in kvpRange_kV:
+        print(kvp)
+        for AddedFiltration_mmAl in AddedFiltrationRange_mmAl:
+            for AddedFiltration_mmCu in AddedFiltrationRange_mmCu:
+
+                filters = [("Al", InherentFiltration_mmAl + AddedFiltration_mmAl), ("Cu", AddedFiltration_mmCu)]
+                s = sp.Spek(kvp=kvp, th=AnodeAngle_deg, dk=1, z=AddedFiltration_mmAir / 10)
+                s.multi_filter(filters)
+
+                res["kVp_kV"].append(kvp)
+                res["InherentFiltration_mmAl"].append(InherentFiltration_mmAl)
+                res["AddedFiltration_mmCu"].append(AddedFiltration_mmCu)
+                res["AddedFiltration_mmAl"].append(AddedFiltration_mmAl)
+                res["AddedFiltration_mmAir"].append(AddedFiltration_mmAir)
+                res["DeviceModel"].append(DeviceModel)
+                res["AcquisitionPlane"].append(AcquisitionPlane)
+                res["AnodeAngle_deg"].append(AnodeAngle_deg)
+                res["HVL_mmAl"].append(s.get_hvl())
+                res["LabName"].append(LabName)
+                res["MeasurementDate"].append(MeasurementDate)
+
+    pd.DataFrame(res).to_csv(ResultsPath / FileName)
+
+
+kvp_min = 25
+kvp_max = 175
+
+
+# Allura Clarity plane A U104 20180412
```

Comment:
I feel like these rows should not be kept in the file. They feel like a special case for running the function above. Is that correct? In that case I think we should remove it.

same goes for `kvp_min` and `kvp_max` above

Reply:
removed